src/m365/spfx/commands/project/project-upgrade/upgrade-1.15.2.ts

```diff
@@ -53,6 +54,7 @@ module.exports = [
   new FN002023_DEVDEP_microsoft_eslint_config_spfx('1.15.2'),
   new FN002001_DEVDEP_microsoft_sp_build_web('1.15.2'),
   new FN002002_DEVDEP_microsoft_sp_module_interfaces('1.15.2'),
+  new FN002019_DEVDEP_spfx_fast_serve_helpers('1.15.2'),
```

Comment:
Nice work @nicodecleyre.
For the spfx-fast-serve-helpers, let's use the latest patch version which is `1.15.4`